providers/terraform-provider-csbmajorengineversion/csbmajorengineversion/datasource_majorengineversion_test.go

```diff
@@ -73,6 +77,13 @@ var _ = Describe("Provider", func() {
 		Entry("mysql 8.0.32", "mysql", "8.0.32", "8.0", ""),
 		Entry("no engine", "", "", "8.0.32", `Error: expected "engine" to not be an empty string`),
 		Entry("no engine version", "mysql", "", "", `Error: expected "engine_version" to not be an empty string`),
+		Entry(
+			"aurora-postgresql 8.0.postgresql_aurora.3.02.0",
+			"aurora-postgresql",
+			"8.0.postgresql_aurora.3.02.0",
```

Comment:
Hmmm, this is not a real postgresql aurora version, this is mysql which is completely different. For postgres, Aurora uses the same versions as plain postresql

Reply:
Yes, exactly. I wanted to get the error `invalid parameter combination`. I will use an engine version closer to reality. Thank you!

Reply:
Thanks for the explanation! LGTM